PayPalWebPayments/src/main/java/com/paypal/android/paypalwebpayments/PayPalWebCheckoutClient.kt

```diff
@@ -5,7 +5,6 @@ import android.content.Intent
 import android.util.Log
 import androidx.activity.ComponentActivity
 import com.paypal.android.corepayments.CoreConfig
-import com.paypal.android.corepayments.PayPalSDKError
```

Comment:
How come you don't need this now?

Reply:
Looks like this import was only used in a function declaration e.g.

```kotlin
private fun notifyWebCheckoutFailure(error: PayPalSDKError, orderId: String?) {}
```

We removed those methods and the import got flagged by `detekt` as no longer needed.

Reply:
Yup since `PayPalWebCheckoutError` is no longer explicitly referenced in this file, we're able to skip importing it.